flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -611,6 +590,10 @@ public long getNumberOfRestarts() {
         return numberOfRestartsCounter.getCount();
     }
 
+    public int getVerticesFinished() {
```

Comment:
nit: `getFinishedVertices`?

Reply:
I also think `getFinishedVertices` would be more nature, but a bit concern here is that the variable to get is name by `verticesFinished`, should we keeps this method to be a getter method for that variable ?

Reply:
I'd also rename the variable.